elyra/pipeline/processor_kfp.py

```diff
@@ -37,6 +38,30 @@ class KfpPipelineProcessor(PipelineProcessor):
     def type(self):
         return self._type
 
+    def get_auth_session_cookie(self, url, api_endpoint_username, api_endpoint_password):
+        get_response = requests.get(url)
+
+        # dex authentication request to kfp will look like '/dex/auth/local?req=REQ_VALUE'
+        if 'auth' in get_response.url:
+            cookie_auth_key = 'authservice_session'
+            credentials = {'login': api_endpoint_username, 'password': api_endpoint_password}
+
+            # Redirect to post authentication request
+            post_response = requests.post(get_response.url, data=credentials)
+            cookie_auth_value = post_response.cookies.get(cookie_auth_key)
+
+            if cookie_auth_value is not None:
+                return cookie_auth_key + '=' + cookie_auth_value
+            else:
```

Comment:
This can be removed.

Reply:
In my use case the cookie is found in the last item of response history, for some reason it wasn't carried over. Maybe I should try using requests.Session instead?

Reply:
Sorry, I meant the `else` statement can be removed and its block unindented since the `if` can only return.